index.html

```diff
@@ -8,15 +8,39 @@
     <title>Awesome Books</title>
 </head>
```

Comment:
- [ ] [OPTIONAL] The readme you have added is not very descriptive. You can add details about your project in it.